src/promptflow/promptflow/_cli/_pf_azure/entry.py

```diff
@@ -113,13 +115,27 @@ def entry(argv):
         setup_user_agent_to_operation_context(args.user_agent)
     logger = get_telemetry_logger()
     custom_dimensions = _get_workspace_info(args)
-    with log_activity(
-        logger,
-        _get_cli_activity_name(cli=prog, args=args),
-        activity_type=ActivityType.PUBLICAPI,
-        custom_dimensions=custom_dimensions,
-    ):
-        run_command(args)
+    activity_name = _get_cli_activity_name(cli=prog, args=args)
+    try:
+        with log_activity(
+                logger,
+                activity_name,
+                activity_type=ActivityType.PUBLICAPI,
+                custom_dimensions=custom_dimensions,
+        ):
+            run_command(args)
```

Comment:
pfs command entry may also need this. 
![image](https://github.com/microsoft/promptflow/assets/47586720/63c00bed-be43-4152-92d0-f3457184e2f0)

Reply:
done.